managedtenants/schemas/metadata.schema.yaml

```diff
@@ -82,6 +82,41 @@ properties:
   pullSecret:
     type: string
     format: printable
+    description: "'pullSecret' is deprecated for now. Please use the 'secrets' and 'pullSecretName' fields instead."
```

Comment:
Seems like there's an incoherence between this file and the zz_metadata_schema_generated.md.

The `pullSecret` field here has a description but the one in `zz_metadata_schema_generated.md` doesn't have one.

You can checkout the `make generate` command which will automatically update the zz_metadata_schema_generated.md as per this metadata.schema.yaml :)